cs/unittest/RunTests-cmake.tt

```diff
@@ -0,0 +1,200 @@
+<#@ template debug="false" hostspecific="True" language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.IO" #>
+<#@ import namespace="System.Globalization" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Text.RegularExpressions" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ assembly name="Newtonsoft.Json.dll" #>
+<#@ import namespace="Newtonsoft.Json" #>
+<#@ output extension=".cs" #>
+
+<#
+
+var testRoot = Path.Combine(Path.GetDirectoryName(Host.TemplateFile), @"..\..\test");
+
+List<Test> tests;
+using (StreamReader sr = File.OpenText(Path.Combine(testRoot, "runtests.AUTOGEN.json")))
+{
+    tests = JsonConvert.DeserializeObject<List<Test>>(sr.ReadToEnd());
+}
+
+var skipList = new[] { 13, 32, 39, 258, 40, 259, 41, 260, 59, 60, 61, 66, 68, 90,
```

Comment:
having two files with skip list will be interesting in the long term, and a bit confusing

Reply:
Yeah agreed. Chatted with jacob and have an idea how to merge the files

Reply:
the newtonsoft dependency could be refactored out of the tt file, this might imply rewriting how this test file gets called